src/librustc_borrowck/diagnostics.rs

```diff
@@ -314,6 +314,90 @@ let c = &i; // still ok!
 ```
 "##,
 
+E0506: r##"
+This error occurs when an attempt is made to assign to a borrowed value.
+
+For example:
```

Comment:
It should "Example of erroneous code". Discussion here: https://github.com/rust-lang/rust/pull/32989#discussion_r60399703